src/screens/Episode/helpers/getTime.ts

```diff
@@ -0,0 +1,29 @@
+import * as dateFns from 'date-fns'
+import { UNIX_TIME_START } from '../common'
+import { DateFormatType, getFormattedDate } from 'src/helpers'
+
+const checkHourExist = (hours: string): boolean => {
+  return hours === '00' ? false : true
+}
+
+const getTime = (secondsAsNumber: number): string => {
```

Comment:
It's very confused argument name. Let's name it `timestamp`

```suggestion
const getPlayerTime= (timestamp: number): string => {
```